cluster-stamp.json

```diff
@@ -40,7 +40,7 @@
             }
         },
         "location": {
-            "defaultValue": "eastus2",
+            "defaultValue": "[resourceGroup().location]",
```

Comment:
Forgot one thing.  One reason we hardcoded this was to ensure we showed the intential parity between this location and our paired region in `geoRedundancyLocation` -- if we change this to have a "by convention" here for the default Value, the paired hard-coded geoRedundancyLocation would no longer make sense for all but one region.  Maybe this PR could make `geoRedundancyLocation` a param without a default value and force the user to set that one?

Reply:
@neilpeterson -- wanted to make sure you saw this still in the chatter. What would be a good solution for this?

Reply:
Thanks Chad. I think I am tracking some of the intricacies here, but perhaps we should talk it out.